README.md

```diff
@@ -5,164 +5,134 @@
-### Commit your code and submit your module for review
-
-1.  Before you commit any code, review [Contributing to the IBM Cloud Terraform modules project](https://terraform-ibm-modules.github.io/documentation/#/contribute-module) in the project documentation.
-1.  Create a pull request for review.
-
-### Post-merge steps
-
-After the first PR for your module is merged, follow these post-merge steps:
-
-1.  Create a PR to enable the upgrade test by removing the `t.Skip` line in `tests/pr_test.go`.
-
+## Summary
+
+This repository contains WaziaaS deployable architecture solutions that help provision VPC landing zones and interconnect them. The below solutions are available and can be deployed with terraform.
+
+Two solutions are offered:
+1. [Quickstart Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/main/solutions/quickstart)
+    This pattern deploys the following infrastructure:
+    - Workload VPC with Wazi as a service VSI.
+    - Uses Floating IP addresses for access through the public internet.
+2. [Standard Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/main/solutions/standard)
+    This pattern deploys the following infrastructure:
+    - Separate VPC for edge.
+    - Separate VPC for workloads.
+    - Virtual Server Instances for every subnet.
+    - A resource group for cloud services and for each VPC.
+    - Cloud Object Storage instances for flow logs and Activity Tracker.
+    - Encryption keys in a Key Protect instance.
+    - A edge and workload VPC connected by a transit gateway.
+    - All necessary networking rules to allow communication.
+    - Virtual Private Endpoint (VPE) for Cloud Object Storage in each VPC.
+    - A client-to-site VPN gateway in the edge VPC.
+    - A jump server Bastion host VSI in the edge VPC without floating IP.
+    - A site-to-site VPN in the workload VPC.
+    - Wazi as a Service VSI in the workload VPC.
 <!-- Remove the content in this previous H2 heading -->
-## Reference architectures
-
-<!--
-Add links to any reference architectures for this module.
-(Usually in the `/reference-architectures` directory.)
-See "Reference architecture" in Authoring Guidelines in the public documentation at
-https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=reference-architecture
--->
-
-## Usage
 
-<!--
-Add an example of the use of the module in the following code block.
+<!-- BEGIN OVERVIEW HOOK -->
+## Overview
+* [terraform-ibm-zvi](#terraform-ibm-zvsi)
+* [Variations](./solutions)
+    * [Quickstart](./solutions/quickstart)
+    * [Standard](./solutions/standard)
+* [Contributing](#contributing)
+<!-- END OVERVIEW HOOK -->
 
-Use real values instead of "var.<var_name>" or other placeholder values
-unless real values don't help users know what to change.
--->
-
-```hcl
-
-```
+## Reference architectures
+- [Quickstart Variation](https://github.com/terraform-ibm-modules/terraform-ibm-zvsi/tree/main/solutions/quickstart)
```

Comment:
Yes both are pointing to the same place since I followed the template [module link](https://github.com/terraform-ibm-modules/terraform-ibm-module-template/blob/main/README.md)